### ISSUE TITLE
Cannot connect to mailchimp

### ISSUE THREAD

**Issue by lisastag** [ibericode/mailchimp-for-wordpress#372]
I have been using the mailchimp plug in successfully for several months.  I am now getting an error message that mailchimp cannot connect.  I generated a new API key but am still getting the same error.  I have the latest version of WordPress and the mailchimp-for-wordpress app.  The website is natureguys.org.  

![mailchimperror](https://cloud.githubusercontent.com/assets/22618525/19078248/7db02c84-8a1d-11e6-8138-e646e573a348.JPG)

**Comment by dannyvankooten** [comment]
Hi @lisastag ,

Unfortunately, that error would indicate something fails when trying to connect to MailChimp. This is not an error in the plugin but can be anything in your server or between your server and the MailChimp server.

Please refer to our [KB article on solving connectivity issues](https://mc4wp.com/kb/solving-connectivity-issues/) for a few hints on where to look for the solution.

**Comment by lisastag** [comment]
Thank you for your reply.  I was able to insert a mailchimp form directly
into my wordpress site which solved my problem and so I am no longer using
your plug in.  I did look at your article on connectivity and I honestly
believe that the error is somewhere in the plug in.  If it helps, I noticed
in the log that I began having the problem immediately after a signup
failed because the email address was already on our list (see below).

Best of luck, Lisa

[2016-09-23 22:02:06] WARNING: Form 91 > *************rjr@gmail.com is
already subscribed to the selected list(s)
[2016-10-04 12:31:09] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 12:31:26] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 12:34:00] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 13:15:24] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 13:18:09] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 13:58:41] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 14:18:59] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 20:22:46] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
[2016-10-04 20:22:52] ERROR: Form 91 > MailChimp API error: 503 Service
Unavailable.
